stacker_blueprints/dynamodb.py

```diff
@@ -0,0 +1,152 @@
+from stacker.blueprints.base import Blueprint
```

Comment:
I'm not badass enough yet to have an edit button on other people's comments.  :D